heft-plugins/heft-lint-plugin/src/test/__snapshots__/snapshot.test.ts.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`formatAsSARIF should correctly format ESLint results into SARIF log 1`] = `
+Object {
+  "$schema": "http://json.schemastore.org/sarif-2.1.0-rtm.5",
+  "runs": Array [
+    Object {
+      "artifacts": Array [
+        Object {
+          "location": Object {
+            "uri": "file:///C:/Users/ating/Work/rushstack-fork/rushstack/heft-plugins/heft-lint-plugin/src/file1.ts",
```

Comment:
This absolute path isn't portable. Also seems not right for CI. Is this really what the SARIF format wants?

If this really is right, you may need to mock the node URL package to generate a path that's portable in snapshots.

Reply:
Absolute path now replaced with relative path. Function previously used in `formatEslintResultsAsSARIFf` to set url would always retrieve absolute path. Fixed to now always use relative path.